<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="del1-lhp-N01966" ignored="0" name="Opera Test" tests="9" failures="4" timestamp="2020-08-03T01:37:54 IST" time="43.42" errors="0">
  <testcase name="loginPageTitleTest" time="0.04" classname="com.sohambko.qa.testcases.LoginPageTest"/>
  <testcase name="loginPageTitleTest" time="0.728" classname="com.sohambko.qa.testcases.LoginPageTest"/>
  <testcase name="crmLogoImageTest" time="0.245" classname="com.sohambko.qa.testcases.LoginPageTest"/>
  <testcase name="crmLogoImageTest" time="0.68" classname="com.sohambko.qa.testcases.LoginPageTest"/>
  <testcase name="loginTest" time="24.52" classname="com.sohambko.qa.testcases.LoginPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.sohambko.qa.util.ElementUtil.doSendKeys(ElementUtil.java:50)
at com.sohambko.qa.pages.LoginPage.login(LoginPage.java:46)
at com.sohambko.qa.testcases.LoginPageTest.loginTest(LoginPageTest.java:31)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="24.465" classname="com.sohambko.qa.testcases.LoginPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.sohambko.qa.util.ElementUtil.doSendKeys(ElementUtil.java:50)
at com.sohambko.qa.pages.LoginPage.login(LoginPage.java:46)
at com.sohambko.qa.testcases.LoginPageTest.loginTest(LoginPageTest.java:31)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginPageTitleTest" time="0.032" classname="com.sohambko.qa.testcases.LoginPageTest"/>
  <testcase name="crmLogoImageTest" time="0.153" classname="com.sohambko.qa.testcases.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.sohambko.qa.testcases.LoginPageTest.crmLogoImageTest(LoginPageTest.java:25)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- crmLogoImageTest -->
  <testcase name="loginTest" time="25.19" classname="com.sohambko.qa.testcases.LoginPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.sohambko.qa.util.ElementUtil.doSendKeys(ElementUtil.java:50)
at com.sohambko.qa.pages.LoginPage.login(LoginPage.java:46)
at com.sohambko.qa.testcases.LoginPageTest.loginTest(LoginPageTest.java:31)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Opera Test -->
